### PR TITLE
STOC and netcdf improvements

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

**Comment by bgruening** [comment]
@yvanlebras I think you need to bump the tool version.

**Comment by yvanlebras** [comment]
Oh, thank you Björn! Nice catch! Done !!!!!

File: info_file.txt [hunk 1]
```diff
@@ -0,0 +1,84 @@
+netcdf dataset_2 {
```
Comment (bgruening) [inline info_file.txt:1]: @yvanlebras the two news files, for what are they good for? They are not used in this PR afaik.

Comment (yvanlebras) [reply info_file.txt]: hmmm really sorry, don't know how these 2 files (test-data from the nectdf GH branch) are arrive here ... AGAIN.... TTHHHHAAAAANNK   YYOUUUUUU you and your eagle eyes ;)

**Pull request merged**